test/tests/selftest/pbs_default_timeout.py

```diff
@@ -36,32 +36,57 @@
 # trademark licensing policies.
 
 from tests.selftest import *
+from ptl.utils.plugins.ptl_test_runner import TimeOut
 
 
 class TestDefaultTimeout(TestSelf):
     """
     Test suite to verify increase default testcase timeout working properly.
     """
 
+    @staticmethod
+    def common_sleep_function(sleep_val):
```

Comment:
why did you need to create this function? couldn't you have just called time.sleep() ?

Reply:
@agrawalravi90 , Initially i did the same but codacy giving error "Method could be a function". to incorporate it i make static common function and called in every test.

Reply:
no, i meant to say that you don't need this function altogether, you can just call time.sleep() instead of calling this function in your code